### PR TITLE
Metrics anomaly detection fix

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

- Metrics anomaly detection fix
- Correcting inconsistencies and missing info

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-541

### CODE REVIEW

File: docs/alerts/monitors/create-monitor.md [hunk 1]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
-
-* **Up**. Only get alerted if there is an abnormal *increase* in the tracked key indicator. 
-* **Down**. Only get alerted if there is an abnormal *decrease* in the tracked key indicator.
-* **Both**. Get alerted if there is *any* abnormality in the data whether an increase or a decrease.
+#### Metrics
 
+Metrics monitors can have up to 6 queries. If you're providing multiple metrics queries, use the letter labels to reference a query row. The monitor will automatically detect the query that triggers your alert, and will mark that row with a notification bell icon. See [Joined metrics queries](/docs/metrics/metrics-queries/metrics-explorer/#join-metric-queries) for details.<br/><img src={useBaseUrl('img/alerts/monitors/metrics-monitor-query-row.png')} alt="Screenshot of the 'New Monitor' setup page in Sumo Logic, showing the Trigger Conditions section. Metrics is selected as the Monitor Type and Static as the Detection Method. The query includes two metrics: CPU_Sys and CPU_User, with an alert condition combining both metrics (#B + #C). A bell icon is highlighted on the left side." style={{border: '1px solid gray'}} width="700"/>
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:125]:
```suggestion
Metrics monitors can have up to 6 queries. If you're providing multiple metrics queries, use the letter labels to reference a query row. The monitor will automatically detect the query that triggers your alert, and will mark that row with a notification bell icon. See [Join metrics queries](/docs/metrics/metrics-queries/metrics-explorer/#join-metric-queries) for details.<br/><img src={useBaseUrl('img/alerts/monitors/metrics-monitor-query-row.png')} alt="Screenshot of the 'New Monitor' setup page in Sumo Logic, showing the Trigger Conditions section. Metrics is selected as the Monitor Type and Static as the Detection Method. The query includes two metrics: CPU_Sys and CPU_User, with an alert condition combining both metrics (#B + #C). A bell icon is highlighted on the left side." style={{border: '1px solid gray'}} width="700"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 2]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
+For example, if your monitor evaluates the last 1 hour, you can set a resolution window of 15 minutes. Once the resolution window is continuously satisfied for 15 minutes, the alert will resolve automatically.<br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-2.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>
+
 #### Static detection method
 
-**Example: Log Trigger Type - Critical and Warning**  
+**Example: Logs - Static - Critical and Warning**  
 
 <img src={useBaseUrl('img/alerts/monitors/logs-trigger-type.png')} alt="logs trigger type.png" style={{border: '1px solid gray'}} width="600"/>
 
 `Alert when returned row count is <threshold type> <threshold> within <time range>`
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:168]: To match the screenshot, should this be `Alert when result is <threshold type> <threshold> within <time range>`?

File: docs/alerts/monitors/create-monitor.md [hunk 3]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
 
-For Metrics monitors, you can choose to recover based on a single data point below the threshold, or all data points below the threshold.<br/><img src={useBaseUrl('img/alerts/monitors/metricsmonitor.png')} alt="Monitors" style={{border: '1px solid gray'}} width="600" />
+For recovery, Sumo Logic will automatically resolves the incident when the resolution condition is satisfied.
 
 #### Anomaly detection method
 
-**Example: Logs Trigger Type - Critical**
+**Example: Logs - Anomaly - Critical**
 
 <img src={useBaseUrl('img/alerts/monitors/monitor-anomaly-logs.png')} alt="Monitor anomaly logs" style={{border: '1px solid gray'}} width="600" />
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:197]: `Alert when anomaly count is at least <count> (max. 1) at any time within <time range>'

Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:197]:
```suggestion

`Alert when anomaly count is at least <count> (max. 1) at any time within <time range>'

```

File: docs/alerts/monitors/create-monitor.md [hunk 4]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
 
 `Alert when missing data within <time range>`
 
 | Parameter | Description |
 |:--|:--|
-| Time range | The time span of data to evaluate. Select either 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 3 hours, 6 hours, 12 hours, or 24 hours. |
+| `<time range>` | The time span of data to evaluate (values range from 5 minutes to 24 hours). |
 
-**Recover**
+For recovery, Sumo Logic will automatically resolves the incident when the resolution condition is satisfied.
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:232]:
```suggestion
For recovery, Sumo Logic will automatically resolve the incident when the resolution condition is satisfied.
```

File: docs/alerts/monitors/create-monitor.md [hunk 5]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
+For example, if your monitor evaluates the last 1 hour, you can set a resolution window of 15 minutes. Once the resolution window is continuously satisfied for 15 minutes, the alert will resolve automatically.<br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-2.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>
 
 #### Prerequisites
 
-To fully leverage AI-driven alerts for metrics monitors, you'll need:
+To fully leverage metrics monitor alerts, you'll need:
 
-* **Automation Service**. Required for linking playbooks to metrics-based monitors.
-* **Metrics data**. Our anomaly detection uses at least 30 days of your Sumo Logic metrics data history to establish baseline of the metrics signal and the underlying system behavior.
+* **Automation Service**. Required for linking playbooks to metrics-based monitors ([learn more](#anomaly)).
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:247]:
```suggestion
* **Automation Service**. Required for linking playbooks to metrics-based monitors ([learn more](#automated-playbooks)).
```

File: docs/alerts/monitors/create-monitor.md [hunk 6]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
+For example, you want to be alerted when the CPU usage is over 60% `at all times` within a 5-minute window. If you set the count to 3, this means that you will only get an alert if you have at least 3 data points showing CPU usage above 60% within that 5-minute window. If you only have 2 data points, even if both of them show CPU usage above 60%, you won't get an alert.
 
-This setting affects both the alert generation logic and the alert recovery logic.
+**Example: Metrics - Static - Missing Data**
 
-![metrics alert datapoints.png](/img/alerts/monitors/minimum-datapoints.png)
+<img src={useBaseUrl('img/alerts/monitors/metrics-static-missing.png')} alt="Metrics static missing data" style={{border: '1px solid gray'}} width="600" />
 
-`Alert and recovery require a minimum of <Count> data points for "at all times" evaluation windows`
+`Alert when missing data <occurrence type> for <time range>`
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:294]:
```suggestion
`Alert when missing data <occurrence type> within <time range>`
```

File: docs/alerts/monitors/create-monitor.md [hunk 7]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
-:::note
-This setting only works when you choose `at all times within` as the type of occurrence for the alert.  
+:::info Limitations
+Anomaly detection applies to one time series at a time. All metrics anomaly monitor trigger queries must have aggregation applied at the end of the query before detection.
 :::
 
+**Example: Metrics - Anomaly - Critical**
+
+<img src={useBaseUrl('img/alerts/monitors/metrics-anomaly-critical.png')} alt="metrics-anomaly-critical" style={{border: '1px solid gray'}} width="600" />
+
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:310]:
```suggestion

`Alert when anomaly count is at least <count> (max. 5) at any time within <time range>`

```

File: docs/alerts/monitors/create-monitor.md [hunk 8]
```diff
@@ -108,242 +116,244 @@ For guidance on optimizing scan costs when using Flex Pricing, refer to the [FAQ
...
+<img src={useBaseUrl('img/alerts/monitors/metrics-anomaly-missing.png')} alt="metrics-anomaly-missing" style={{border: '1px solid gray'}} width="600" />
+
+`Alert when missing data within <time range>`
+
+| Parameter | Description |
+|:--|:--|
+| `<time range>` | The duration of time to evaluate (values range from 5 minutes to 24 hours). |
+
+For recovery, Sumo Logic will automatically resolves the incident when the resolution condition is satisfied.
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:328]:
```suggestion
For recovery, Sumo Logic will automatically resolve the incident when the resolution condition is satisfied.
```

File: docs/alerts/monitors/create-monitor.md (hunk 2 above)
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:168]:
```suggestion
`Alert when result is <threshold type> <threshold> within <time range>`
```

File: docs/alerts/monitors/create-monitor.md (hunk 3 above)
Comment (jpipkin1) [reply docs/alerts/monitors/create-monitor.md:197]: You need to add this line under the screenshot o be consistent with the other examples.

**Review by jpipkin1 — COMMENTED** [review]
Wow, @kimsauce, that's an amazing amount of work! My head is spinning just trying to absorb all the information you added. It looks really good.

**Review by jpipkin1 — APPROVED** [review]

**Comment by kimsauce** [comment]
> Wow, @kimsauce, that's an amazing amount of work! My head is spinning just trying to absorb all the information you added. It looks really good.

thank you for taking the time to carefully review it! I missed a lot of stuff. appreciate your help.

**Pull request merged**